app/controllers/render-tree.js

```diff
@@ -2,16 +2,44 @@ import Ember from "ember";
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
 import debounceComputed from "ember-inspector/computed/debounce";
 
-const { computed, isEmpty, Controller, inject: { controller } } = Ember;
+const { computed, isEmpty, Controller, inject: { controller, service } } = Ember;
 const { and, equal, filter } = computed;
 const get = Ember.get;
+let LOCAL_STORAGE_SUPPORTED;
+try {
+  LOCAL_STORAGE_SUPPORTED = !!window.localStorage;
+} catch (e) {
+  // Security setting in chrome that disables storage for third party
+  // throws an error when `localStorage` is accessed.
+  LOCAL_STORAGE_SUPPORTED = false;
+}
```

Comment:
At this time `ember-inspector` doesn't work in Safari, but if it did, this check is not sufficient.
When you use Safari in Private mode, it appears that you have `localStorage` but you don't.
Instead, something like this would work:
```
    try {
      localStorage.setItem('test', 0);
      localStorage.removeItem('test');
      return true;
    } catch(_) {
      return false;
    }
```